### PR TITLE
Update the content of Package.md and README.md files & replace icon with high resolution image

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Related to: https://github.com/ballerina-platform/ballerina-standard-library/issues/2657

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Package.md [hunk 1]
```diff
@@ -1,14 +1,451 @@
 ## Package Overview
 
-The `jdbc` library is one of the external library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+This module provides the functionality that is required to access and manipulate data stored in any type of relational database,
```
Comment (niveathika) [inline ballerina/Package.md:3]:
```suggestion
This package provides the functionality that is required to access and manipulate data stored in any type of relational database,
```
Check in other places as well

Comment (kaneeldias) [reply ballerina/Package.md]: Fixed in [9deb1b5](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/347/commits/9deb1b5ac0037560497ae32a399ccebaa2f2d8c2)

File: README.md [hunk 2]
```diff
@@ -1,35 +1,458 @@
...
+[![Trivy](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/actions/workflows/trivy-scan.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-java.jdbc.svg)](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/commits/master)
+[![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/java.jdbc.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fjava.jdbc)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-java.jdbc/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-java.jdbc)
+
+This package provides the functionality that is required to access and manipulate data stored in any type of relational database,
```
Comment (daneshk) [inline README.md:9]:
```suggestion
This library provides the functionality that is required to access and manipulate data stored in any type of relational database,
```

File: ballerina/Module.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Overview
 
-This module provides the functionality that is required to access and manipulate data stored in any type of relational database,
+This package provides the functionality that is required to access and manipulate data stored in any type of relational database,
```
Comment (daneshk) [inline ballerina/Module.md:3]:
```suggestion
This module provides the functionality that is required to access and manipulate data stored in any type of relational database,
```

Comment (daneshk) [reply ballerina/Module.md:3]: shall we revert the change in module.md

Comment (kaneeldias) [reply ballerina/Module.md]: Fixed in [e697738](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/347/commits/e697738db73e0796ce31139cbc8ac128975d0716)

File: README.md (hunk 2 above)
Comment (kaneeldias) [reply README.md]: Fixed in [e697738](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/347/commits/e697738db73e0796ce31139cbc8ac128975d0716)

**Review by daneshk — APPROVED** [review]

**Pull request merged**